labman/gui/handlers/process_handlers/quantification_process.py

```diff
@@ -55,22 +56,22 @@ def post(self):
 
                     # cache the sample compositions to avoid extra intermediate
                     # queries
-                    if isinstance(comp, LibraryPrep16SComposition):
+                    if isinstance(comp, (LibraryPrep16SComposition,
```

Comment:
I think I may have misinformed you when I give you the information from the GDNAComposition. It should be:
```
comp.sample_composition
```